src/iop/basecurve.c

```diff
@@ -1792,22 +1792,15 @@ static gboolean _scrolled(GtkWidget *widget, GdkEventScroll *event, gpointer use
 
   if(c->selected < 0) return TRUE;
 
-  int handled = 0;
-  float dy = 0.0f;
-  if(event->direction == GDK_SCROLL_UP)
-  {
-    handled = 1;
-    dy = BASECURVE_DEFAULT_STEP;
-  }
-  if(event->direction == GDK_SCROLL_DOWN)
+  gdouble delta_x, delta_y;
```

Comment:
> The current (inverted) behaviour is on purpose as it's what hanatos prefers. Please keep it that way in this PR. Re-evaluating that decision is something for another day.

Thank you for clarification! I flipped it back.

One question is what to do about horizontal scrolls. Prior to this PR, they were just plain ignored. Now they are handled (presumably only from trackpad/trackpoint/touch) and their direction is inverted (to be consistent with vertical scrolling). Would it be better just to ignore them, such that scrolling would be used to fine-tune the y-value of points without messing up the x-axis? Alternately, some keyboard modifier (meta/alt?) could be used to switch from y-only scrolling to x and y scrolling?